extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/SimpleSpanProcessorConfig.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.opentelemetry.runtime.config.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithName;
+
+@ConfigGroup
+public interface SimpleSpanProcessorConfig {
+
+    /**
+     * Do we use simple or batch, default is batch,
+     * since this value is false by default.
+     */
+    @WithDefault("false")
+    boolean enabled();
+
+    /**
+     * Whether unsampled spans should be exported.
+     * <p>
+     * Default is `false`.
+     */
+    @WithName("export.unsampled.spans")
```

Comment:
Please remove this property. For consistency, sampling should only be done in the sampler.

Reply:
Why?
If they have it on the SSP builder, I would leave it here as well ...

Reply:
because it's not included in the autoconfiguration in the OTel side settings and it is scope creep.

Reply:
Scope creep? :-)